dockerfiles/contracts-ci-linux/Dockerfile

```diff
@@ -27,8 +27,19 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN set -eux; \
 	apt-get -y update && \
 	apt-get install -y --no-install-recommends \
-		zlib1g-dev binaryen npm wabt llvm-dev && \
+		zlib1g-dev npm wabt llvm-dev && \
 	npm install --ignore-scripts -g yarn && \
+# binaryen is needed by cargo-contract for optimizing Wasm files
+# we fetch the latest version number from the GitHub API and use that release
+	curl --silent https://api.github.com/repos/WebAssembly/binaryen/releases/latest | \
+		egrep --only-matching 'https://github.com/WebAssembly/binaryen/releases/download/version_[0-9]+/binaryen-version_[0-9]+-x86_64-linux.tar.gz' | \
+		head -n1 | \
+		xargs curl -L -O && \
+	tar -xvzf binaryen-version_*-x86_64-linux.tar.gz  && \
```

Comment:
I hope you know what you are doing going with the latest. Are they stable enough to use in prod?
I'd vote for pinned version, but we do not have a strict policy for it.